### PR TITLE
PMM-4176 QAN for PostgreSQL.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-385/

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -964,6 +1039,20 @@ service Agents {
...
   }
+  // AddQANPostgreSQLPgStatementsAgent adds PostgreSQL Pg Stat Statements QAN Agent.
+  rpc AddQANPostgreSQLPgStatementsAgent(AddQANPostgreSQLPgStatementsAgentRequest) returns (AddQANPostgreSQLPgStatementsAgentResponse) {
+    option (google.api.http) = {
+      post: "/v0/inventory/Agents/AddQANPostgreSQLPgStatementsAgent"
+      body: "*"
+    };
+  }
+  // ChangeQANPostgreSQLPgStatementsAgent changes PostgreSQL Pg Stat Statements QAN Agent.
+  rpc ChangeQANPostgreSQLPgStatementsAgent(ChangeQANPostgreSQLPgStatementsAgentRequest) returns (ChangeQANPostgreSQLPgStatementsAgentResponse) {
```
Comment (AlekSi) [inline api/inventorypb/agents.proto]: Please make PR to pmm-managed with stubs for those methods so we could update without complication failures.

Comment (BupycHuk) [reply api/inventorypb/agents.proto]: ok, please, don't merge until I make PR :)

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-385/

**Pull request merged**